guide/src/ref/packet-log.md

```diff
@@ -1,37 +1,167 @@
 # Packet Log
 
-The packet log contains a binary file that contains a compressed record of all packets processed by Rattan and a flow description file. It is designed to store what happened during the experiment with low overhead, enabling post-analysis and debugging.
+The Rattan packet log contains the following binary files:
+
+- A `.flow` file, which contains a basic describe of all (recognized) flows processed by Rattan.
+- A `.rtl` file, which contains a compressed record of the packets in those flows.
+- Optionally, a `.raw` file, which contains the protocol headers of those packets.
+
+It is designed to store what happened during the experiment with low overhead, enabling post-analysis and debugging.
+
+**Notice**:
+Rattan records TCP flows on IPv4 only now. Any other packets that is not recognized as a TCP flow on IPv4 are ignored
+when recording a Rattan packet log, while they are normally processed and forwarded by
+Rattan (e.g., UDP flows).
 
 ## Usage
 
-Using the `--packet-log` option in the CLI will enable the packet log feature and specify the file to store the records. The flow description file will be stored in the same directory with the name `<packet_log>.flows`, where `<packet_log>` is the name of the packet log file.
+### Record a Rattan packet Log
 
-For example, you can run the following command to enable packet logging:
+Two options are involved: `--packet-log` and `--packet-log-mode` . Example usages are:
 
 ```bash
-rattan link --packet-log packet.log
+rattan link --packet-log packet.rtl --packet-log-mode compact-tcp
+rattan run --config CONFIG_FILE --packet-log packet.rtl --packet-log-mode raw-ip
```

Comment:
Should be `<CONFIG_FILE>` not `CONFIG_FILE` if you use this to indicate that user should use their config file names, or you could just use `config.toml`.